src/views/webviews/welcome/welcome.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+header {
+  margin-top: 20px;
+}
+
+header img {
+  width: 25%;
+  height: 25%;
+  height: auto;
+}
+
+.container {
+  display: flex;
+}
+
+.column {
+  flex: 1;
+
+}
+
+.column:first-child {
+  flex: 2;
+}
+
+.column:last-child {
+  margin-left: 50px;
+}
+
+#desc {
+  color: #f6f4f9;
+  margin: 5px;
+  font-size: 1.0em;
+}
+
+#intro p {
+  color: #c2c9d3;
+  line-height: 1.5;
+}
+
+#intro strong {
+  color: #ffffff;
```

Comment:
Is there some type of theme that we can use instead of hardcoding colours?

Reply:
Will set theme once we have more web views and have proper frame work for them